src/ng/interpolate.js

```diff
@@ -338,7 +371,7 @@ function $InterpolateProvider() {
 
               return compute(values);
             } catch (err) {
-              $exceptionHandler($interpolateMinErr.interr(text, err));
+              $exceptionHandler(err);
```

Comment:
Yeah, that's weird, I have no idea why I changed that. Fixed, and the tests that broke with it.